curriculum/getChallenges.js

```diff
@@ -16,10 +16,9 @@ function getChallengesDirForLang(lang) {
 }
 
 function getMetaForBlock(block) {
-  const meta = JSON.parse(
+  return JSON.parse(
     fs.readFileSync(path.resolve(metaDir, `./${block}/meta.json`), 'utf8')
```

Comment:
Shouldn't we be use readFile instead of readFileSync?

Reply:
not sure, should we?

Reply:
It would be non-blocking using the async readFile.

Reply:
Don't we want to wait for that to finish so we're sure the data is there. I dunno, I tried to switch to the async method and it keeps giving me an error - presumably because the data hasn't been retrieved before it tries to access it.

Reply:
The problem seems to be that `getMetaForBlock` is called inside `describe` and `describe` only takes synchronous functions as its arguments.  I think, anyway.

The solution seems to be to do all the async setup first, then run the tests.  Currently it's a mixture.  If we do that then we can potentially save a bit of time by starting puppeteer and sourcing the challenges in parallel.  At the very least it should be easier to reason about.

Reply:
So are you guys thinking this needs to be changed up?